templates/access.html

```diff
@@ -92,6 +93,8 @@ <h4 id="exploit-description" style="margin-top:2px;text-align:center"></h4>
 
 <script src="/gui/js/ability.js"></script>
 <script>
+    var accessAbilities = {{ abilities | sort(attribute='name') | tojson }};
```

Comment:
Nit: should this be `const`?

Reply:
For whatever reason `accessAbilities` is undefined if created with `const` or `let`